.env.example

```diff
@@ -12,6 +12,14 @@
 DATABASE_USER=postgres
 DATABASE_NAME=timeoverflow_development
 
+#RAILS CONFIG
+RAILS_SERVE_STATIC_FILES=true
+RAILS_LOG_TO_STDOUT=true
+RAILS_LOG_LEVEL=debug
+QUEUE_ADAPTER=sidekiq
```

Comment:
@microstudi do we really need this to be configurable via ENV var?

Reply:
I think RAILS_LOG_LEVEL needs to stay. The rest can be hidden/hardcoded. See if you prerer it as they are now